bundles/net.tourbook.cloud/src/net/tourbook/cloud/dropbox/messages.properties

```diff
@@ -11,9 +11,10 @@ Image__Dropbox_Folder       = dropbox-folder.png
 Image__Dropbox_Logo         = dropbox-logo.png
 Image__Dropbox_Parentfolder = dropbox-parentfolder.png
 
-Pref_CloudConnectivity_Dropbox_AccessToken_Label        = Access Token
-Pref_CloudConnectivity_Dropbox_AccessToken_Tooltip      = This is the access token to access your Dropbox account
-Pref_CloudConnectivity_Dropbox_Button_Authorize         = Authorize
-Pref_CloudConnectivity_Dropbox_ExpiresAt_Label          = Expires At
-Pref_CloudConnectivity_Dropbox_RefreshToken_Label       = Refresh Token
-Pref_CloudConnectivity_Dropbox_Tokens_Information_Group = Tokens Information
+Pref_CloudConnectivity_Dropbox_AccessToken_Label   = Access token
+Pref_CloudConnectivity_Dropbox_AccessToken_Tooltip = This is the access token to access your Dropbox account
+Pref_CloudConnectivity_Dropbox_Button_Authorize    = Authorize
+Pref_CloudConnectivity_Dropbox_ExpiresAt_Label     = Expires at
+Pref_CloudConnectivity_Dropbox_RefreshToken_Label  = Refresh token
+Pref_CloudConnectivity_Dropbox_WebPage_Label       = Web page
+Pref_CloudConnectivity_Dropbox_WebPage_Link        = https://www.dropbox.com
```

Comment:
FYI and not an issue.

I'm using a little bit different naming structure for the text keys, a the name of the used control (or other type) is the 3rd part in the name and not at the end. At the end there is e.g. _Tooltip, _Message or _Title, here is an example:

![mt-text-key-naming](https://user-images.githubusercontent.com/1283445/104017567-a0e31100-51b8-11eb-851c-1381f1e96b18.png)

Reply:
I wouldn't mind if you wanted to establish a standard rule such as :
Pref_ClassName_CompositeName_UIType ?

Reply:
I think it is difficult to create a common rule but the text keys are automatically grouped by the first two parts which are separated with an _ (underscrore), this is the reason why images have 2 subsequent underscrores.

In the beginning I started words in the text keys with lowercase but they cannot be navigated easily with the cursor keys (depending on the location in the IDE) so I switched to ProperCase.